README.md

```diff
@@ -1,14 +1,20 @@
 # Cloud Foundry CLI Database Connection Plugin [![Build Status](https://travis-ci.org/18F/cf-db-connect.svg?branch=master)](https://travis-ci.org/18F/cf-db-connect) [![Code Climate](https://codeclimate.com/github/18F/cf-db-connect/badges/gpa.svg)](https://codeclimate.com/github/18F/cf-db-connect)
 
-This plugin makes it easy to connect to your databases in Cloud Foundry from your local machine. This condenses the steps listed in [Accessing Services with SSH](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html) to a single command. Requires Diego architecture with [SSH enabled](https://docs.cloudfoundry.org/running/config-ssh.html). Currently supports (most) service brokers for the following:
+This plugin makes it easy to connect to your databases in Cloud Foundry from your local machine. This condenses the steps listed in [Accessing Services with SSH](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html) to a single command. Requires Diego architecture with [SSH enabled](https://docs.cloudfoundry.org/running/config-ssh.html).
+
+## Support
+
+Currently supports (most) service brokers for the following:
 
 * MySQL
 * PostgreSQL
 
+Doesn't run on Windows [yet](https://github.com/18F/cf-db-connect/issues/13).
+
 ## Local installation
 
 1. Install the Cloud Foundry CLI v6.15.0 or later.
-1. Install the plugin. [TODO]
+1. Download the appropriate binary from [the Releases page](https://github.com/18F/cf-db-connect/releases).
```

Comment:
hmm. we may want to mention that they could copy the url for their system and use that url directly in `cf install-plugin`